java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -166,42 +80,49 @@ public ColumnVector(long nativePointer) {
   public ColumnVector(DType type, long rows, Optional<Long> nullCount,
       DeviceMemoryBuffer dataBuffer, DeviceMemoryBuffer validityBuffer,
       DeviceMemoryBuffer offsetBuffer) {
+    super(ColumnVector.initViewHandle(
```

Comment:
If we are ok with this, it will help reduce possible ways to create a column view. At some point we could combine this viewHandle initialization with off heap as well. Are we ok with doing it this way, for now? This might be leak sensitive imo.

Reply:
It is more leak sensitive and I really don't like it when we leak things.  I would personally deprecate this API and replace it with a factory method that could remove the leak possibility.

Reply:
I have passed this viewHandle now in OffHeapState